src/mesh/checkpoint_io.C

```diff
@@ -38,13 +38,62 @@
 #include "libmesh/node.h"
 #include "libmesh/parallel.h"
 #include "libmesh/partitioner.h"
+#include "libmesh/metis_partitioner.h"
 #include "libmesh/remote_elem.h"
 #include "libmesh/xdr_io.h"
 #include "libmesh/xdr_cxx.h"
 
 namespace libMesh
```

Comment:
The `chunking` helper function should be in an anonymous namespace.

Reply:
done